src/main/java/org/tailormap/api/configuration/dev/PopulateTestData.java

```diff
@@ -978,6 +978,12 @@ public void createAppTestData() throws Exception {
             .setAuthorizationRules(ruleAnonymousRead)
             .setComponents(
                 List.of(
+                    new Component()
+                        .type("SIMPLE_SEARCH")
+                        .config(
+                            new ComponentConfig()
+                                .enabled(true)
+                                .putAdditionalProperty("municipalities", List.of("Utrecht"))),
```

Comment:
We will be using municipality code for the list, so should be `List.of("0344")`, see https://github.com/Tailormap/tailormap-viewer/pull/787